lib/config/rules/rule-helpers/space-before-function-paren-rule.js

```diff
@@ -0,0 +1,9 @@
+// Consumed by space-before-function-paren rule in style.js
```

Comment:
Typically, if it's a really long array of values, or if a rule requires and object with more than a couple key-value pairs. 
Breaking it out like this is just personal styling preference.